app/tranformers/api/v1/appointment_transformer.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::AppointmentTransformer < Api::Current::Transformer
+  class << self
+    def new_keys_mapping
+      { invalid_phone_number: :other,
+        public_hospital_transfer: :other,
+        moved_to_private: :other }.with_indifferent_access
+    end
+
+    def to_response(model)
+      h =  model.attributes.with_indifferent_access
+      h[:cancel_reason] = new_keys_mapping[h[:cancel_reason]]
```

Comment:
Cancel reasons not part of new_keys_mapping will be transformed to nil.

Reply:
Oh ya. Will fix this.